instrumentation/hibernate/hibernate-6.0/spring-testing/src/test/java/spring/jpa/SpringJpaTest.java

```diff
@@ -0,0 +1,387 @@
+                            equalTo(
+                                DB_SYSTEM, DbIncubatingAttributes.DbSystemIncubatingValues.HSQLDB),
+                            equalTo(maybeStable(DB_NAME), "test"),
+                            equalTo(
+                                DbIncubatingAttributes.DB_USER,
+                                emitStableDatabaseSemconv() ? null : "sa"),
+                            equalTo(
+                                DbIncubatingAttributes.DB_CONNECTION_STRING,
+                                emitStableDatabaseSemconv() ? null : "hsqldb:mem:"),
+                            satisfies(
+                                maybeStable(DB_STATEMENT),
+                                val ->
+                                    val.matches(
+                                        "select ([^.]+)\\.id([^,]*),([^.]+)\\.firstName([^,]*),([^.]+)\\.lastName(.*)from Customer(.*)")),
+                            equalTo(maybeStable(DB_OPERATION), "SELECT"),
+                            equalTo(maybeStable(DB_SQL_TABLE), "Customer")),
+                span ->
+                    span.hasName("Transaction.commit")
+                        .hasKind(INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(
+                                AttributeKey.stringKey("hibernate.session_id"),
+                                trace
+                                    .getSpan(1)
+                                    .getAttributes()
+                                    .get(stringKey("hibernate.session_id"))))));
+
+    testing.clearData();
+
+    testing.runWithSpan("parent", () -> repo.save(customer));
+    Long savedId = customer.getId();
+
+    assertThat(customer.getId()).isNotNull();
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("parent")
+                        .hasKind(INTERNAL)
+                        .hasNoParent()
+                        .hasAttributes(Attributes.empty()),
+                span ->
+                    span.hasName("Session.persist spring.jpa.Customer")
+                        .hasKind(INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            satisfies(
+                                AttributeKey.stringKey("hibernate.session_id"),
+                                val -> val.isInstanceOf(String.class))),
+                span ->
+                    span.hasName("CALL test")
+                        .hasKind(CLIENT)
+                        .hasParent(trace.getSpan(1))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(
+                                DB_SYSTEM, DbIncubatingAttributes.DbSystemIncubatingValues.HSQLDB),
```

Comment:
You could also do a static import for this `HSQLDB` which is used in a few places
```suggestion
                                DB_SYSTEM, HSQLDB),
```